sites/public/pages/listings.tsx

```diff
@@ -37,10 +40,21 @@ const closedListings = (listings) => {
 }
 
 export default function ListingsPage(props: ListingsProps) {
+  const { profile } = useContext(AuthContext)
   const pageTitle = `${t("pageTitle.rent")} - ${t("nav.siteTitle")}`
   const metaDescription = t("pageDescription.welcome", { regionName: t("region.name") })
   const metaImage = "" // TODO: replace with hero image
 
+  useEffect(() => {
+    pushGtmEvent<ListingList>({
+      event: "pageView",
+      pageTitle: pageTitle,
```

Comment:
Not sure if it matters but we do translate page titles - so if we wanted to look up listings hits by page title we would need to look up all the language strings

Reply:
This is a good question and I asked Lori for her preference, so waiting to hear back on that. I think it'd be useful to track it.